### ISSUE TITLE
error in kbinxml

### ISSUE THREAD

**Issue by billychan307** [mon/kbinxml#9]
![image](https://github.com/mon/kbinxml/assets/148402428/d9976b05-0ba9-49c6-bb57-2f41f7b51229)

may i know how to fix it?

**Comment by mon** [comment]
This usually means you provided a file that was not kbin, and not xml either, but it could be something else. Can I see a copy of the raw data?

**Comment by billychan307** [comment]
<img width="577" alt="image" src="https://github.com/mon/kbinxml/assets/148402428/d19a6d35-af8e-4271-b274-279890cb9369">
May i know this information is enough for you? because the raw sources code cannot public it.

**Comment by mon** [comment]
You are welcome to email me if you would like, I can keep the file secret. It might be a corrupt kbin, or I may have misunderstood something about how attribute values are encoded. Either way, I'm interested :)

**Comment by billychan307** [comment]
Do you have discord id? :)

**Comment by mon** [comment]
Yes, it's also `mon`. I get about 5 random friend requests a day so I have temporarily unblocked requests, feel free to contact me.

**Comment by billychan307** [comment]
i have send the request, same as my name
